docs/cli/cli-orders.md

```diff
@@ -530,19 +530,33 @@ The harmonize tool allows you to compare data to different generations of satell
 ]
 ```
 
-You may create an order request by calling `tools.json` with `--tools`.
+You may create an order request by calling [`tools-harmonize.json`](https://raw.githubusercontent.com/planetlabs/planet-client-python/main/docs/cli/request-json/tools-harmonize.json) with `--tools`.
 
 ```console
-planet orders request psscene analytic_sr_udm2 --name "Harmonized data" --id 20200925_161029_69_2223 --tools tools.json
+planet orders request psscene analytic_sr_udm2 --name 'Harmonized data' --id 20200925_161029_69_2223 --tools tools-harmonize.json
 ```
 
+## More options
+
 ### STAC Metadata
 
-TODO
+A relatively recent addition to Planet's orders delivery is the inclusion of [SpatioTemporal Asset Catalog](https://stacspec.org/en)
+(STAC) metadata in Orders. This provides a more standard set of JSON fields, that work with a number of 
+[different tools](https://stacindex.org/ecosystem). The CLI `orders request` command currently requests
+STAC metadata by default, as the STAC files are small and often more useful than the default JSON metadata.
+You can easily turn off STAC output request with the `--no-stac` command:
+
+```
+planet orders request PSScene visual --name 'No STAC' --no-stac --id 20220605_124027_64_242b 
+```
+
+Currently this needs to be done for any 'composite' operation, as STAC output from composites is not yet 
```

Comment:
```suggestion
For this current pre-release, STAC output from composites is not yet 
```

Reply:
Rejecting this one, as this is an issue with Planet's API's, so won't be solved when SDK is out of pre-release.